lib/metadata_json_deps/rake_tasks.rake

```diff
@@ -0,0 +1,7 @@
+require 'metadata_json_deps'
```

Comment:
There is already a rake_tasks.rb file so there is no need for this. See [here](https://github.com/puppetlabs/metadata_json_deps/blob/master/lib/metadata_json_deps/rake_tasks.rb).